client/src/pages/Bookings/Bookings.tsx

```diff
@@ -0,0 +1,146 @@
+import { useState, useEffect } from 'react';
+import { Grid, Typography, Paper } from '@mui/material';
+import getRequests from '../../helpers/APICalls/getRequests';
+import updateRequest from '../../helpers/APICalls/updateRequest';
+import { Request } from '../../interface/RequestApiData';
+import Booking from './Booking/Booking';
+import Calendar from './Calendar/Calendar';
+import useStyles from './useStyles';
+import { RequestStatus } from '../../types/RequestStatus';
+
+export default function Sitters(): JSX.Element {
+  const classes = useStyles();
+  const [requests, setRequests] = useState<Request[]>([]);
+  const [pastRequests, setPastRequests] = useState<Request[]>([]);
+
+  useEffect(() => {
+    getRequests().then((data) => {
+      if (data.error) {
+        console.error({ error: data.error.message });
+      } else if (data.success) {
+        const futureRequests: Request[] = [];
+        const pastRequests: Request[] = [];
+        const today = new Date();
+        data.success.requests.map((request) => {
+          if (new Date(request.start) < today) {
+            pastRequests.push(request);
+          } else {
+            futureRequests.push(request);
+          }
+        });
+
+        setRequests(futureRequests);
+        setPastRequests(pastRequests);
+      } else {
+        console.error({ data });
+      }
+    });
+  }, []);
+
+  function handleRequest(requestId: string, status: RequestStatus) {
+    updateRequest(requestId, status).then(() => {
```

Comment:
In case an update has failed, we can update the Snackbar with the appropriate error message